static/js/main.js

```diff
@@ -198,12 +198,6 @@
     for (var i = 0; i < linkEls.length; i++) {
```

Comment:
Personally I'd change this to a `forEach` since it's shorter, but maybe later.

Reply:
In fact it does no obvious performance improvement, so I think this can be changed later :)